packages/server/lib/hooks/connection/post-connection.ts

```diff
@@ -52,17 +52,23 @@ async function execute(createdConnection: RecentlyCreatedConnection, providerNam
 
                 return connection as Connection;
             },
-            proxy: async ({ method, endpoint, data, headers, params }: UserProvidedProxyConfiguration) => {
+            proxy: async ({ method, endpoint, data, headers, params, baseUrlOverride }: UserProvidedProxyConfiguration) => {
```

Comment:
While debugging using `baseUrlOverride` within the post-connection hook I traced it back to it not being passed through here.